package/gluon-client-bridge/files/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -21,14 +21,20 @@ local function configure_client(config, radio, index, suffix)
 
   uci:delete('wireless', name)
 
+  if index == 0 then
+    macaddr = util.generate_mac(0)
+  elseif index == 1 then
+    macaddr = util.generate_mac(3)
+  end
+
```

Comment:
Just change this to `macaddr = util.generate_mac(index * 3)`, so at least when no IBSS is used, a third WLAN adapter can be supported without problems.